grafana/Dockerfile

```diff
@@ -15,6 +15,10 @@ ENV GF_ANALYTICS_REPORTING_ENABLED=false \
     DATABASE_PORT=5432 \
     DATABASE_SSL_MODE=disable
 
+# Scenes-powered Grafana dashboards disabled via feature toggle
+# until grafana/grafana #95209 is resolved
+ENV GF_FEATURE_TOGGLES_DASHBOARDSCENE=false
```

Comment:
@JakobLichterfeld - added this as Links between Dashboards incl. Time Ranges are currently broken in Scenes-powered Dashboards (enabled by default in Grafana 11.3.0) - will keep an eye on fix upstream.